adapters/ix/ixtest/supplemental/dsa-request.json

```diff
@@ -0,0 +1,193 @@
+{
+    "mockBidRequest": {
+      "id": "test-request-id",
+      "imp": [
+        {
+          "id": "test-imp-id",
+          "banner": {
+            "format": [
+              {
+                "w": 300,
+                "h": 250
+              }
+            ]
+          },
+          "ext": {
+            "bidder": {
+              "siteId": "569749"
+            }
+          }
+        }
+      ],
+      "regs": {
+        "ext": {
+          "dsa": {
+            "required": 3,
+            "pubrender": 0,
+            "topub": 2,
```

Comment:
I just want to point out that the fields `required` and `topub` are supposed to be `dsarequired` and `datatopub` respectively.

Reply:
Ah thanks! updated the fields